web/src/test/resources/application-local.properties

```diff
@@ -27,11 +27,11 @@ fpsak.okonomi.oppdrag.send.queueName=LOKAL_TEST_OPPDRAG_SEND
 ray.avstem.data.queueName=LOKAL_TEST_GRENSESNITTAVSTEMMING_SEND
 
 # Database
-defaultDS.url=jdbc:oracle:thin:@//localhost:1521/XE
+defaultDS.url=jdbc:oracle:thin:@//localhost:1522/XEPDB1
```

Comment:
Er det noe grunn til at du endrer det her? Det vil nok ødelegge for alle som kjører fpsak lokalt regner jeg med.

Reply:
Min feil. Lokale tilpasninger som ble. Fjernet nå